### ISSUE TITLE
Crash using container method if socat is not installed

### ISSUE THREAD

**Issue by exarkun** [telepresenceio/telepresence#431]
On macOS CI (which didn't previously know about the socat requirement):

```
E           Looks like there's a bug in our code. Sorry about that!
E           
E           Here's the traceback:
E           
E           Traceback (most recent call last):
E             File "/Users/distiller/project/telepresence/cli.py", line 73, in call_f
E               return f(*args, **kwargs)
E             File "/Users/distiller/project/telepresence/main.py", line 480, in go
E               runner, args
E             File "/Users/distiller/project/telepresence/main.py", line 300, in start_proxy
E               processes, socks_port, ssh = connect(runner, remote_info, args)
E             File "/Users/distiller/project/telepresence/main.py", line 198, in connect
E               ), "TCP4:127.0.0.1:{}".format(ssh.port)
E             File "/Users/distiller/project/telepresence/runner.py", line 104, in popen
E               return self.launch_command(track, *args, **kwargs)
E             File "/Users/distiller/project/telepresence/runner.py", line 61, in launch_command
E               process = Popen(*args, **kwargs)
E             File "/usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/subprocess.py", line 709, in __init__
E               restore_signals, start_new_session)
E             File "/usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/subprocess.py", line 1344, in _execute_child
E               raise child_exception_type(errno_num, err_msg, err_filename)
E           FileNotFoundError: [Errno 2] No such file or directory: 'socat': 'socat'
```

**Comment by ark3** [comment]
We need to add `socat` to the pile of `require_command(...)` calls in main.py.

**Comment by ark3** [comment]
Fixed in #720.
